internal/controller/state/conditions/conditions.go

```diff
@@ -945,6 +971,17 @@ func NewGatewayInvalidParameters(msg string) Condition {
 	}
 }
 
+// NewGatewayUnsupportedField returns a Condition that indicates the Gateway is accepted but
+// contains a field that is not supported.
+func NewGatewayUnsupportedField(msg string) Condition {
+	return Condition{
+		Type:    string(GatewayConditionUnsupportedField),
+		Status:  metav1.ConditionTrue,
+		Reason:  string(GatewayReasonUnsupportedField),
+		Message: fmt.Sprintf("Gateway is accepted, but some parameters are ignored due to an error: %s", msg),
```

Comment:
As above, this isn't an error:


```suggestion
		Message: fmt.Sprintf("Gateway is accepted, but some parameters are ignored because they are not supported: %s", msg),
```